dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/Standard/Hydraulic/package.order

```diff
@@ -1,2 +1 @@
 HyGov
```

Comment:
Could you correct in this model the base here and put it base PNomTurb :
Modelica.Blocks.Interfaces.RealInput PmRefPu(start = PmRef0Pu) "Reference mechanical power in pu **(base SNom)**"